### PR TITLE
Version one

### PR DESCRIPTION
**Pull request by arashsafaridev**
Initial Version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+```
+~% npm install
+~% npm start
+```
+
+## Install the extension on Chrome/Brave
+
+-   Enter `chrome://extensions/` in Chrome/Brave URL
+-   Activate developer mode
+-   `Load unpacked` then choose the repo
```
Comment (timatron) [inline README.md]: explain this more please, can you do it from the url or have to download first? can we publish it so its a packaged extensions?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
-# chrome-extension-campaign-monitor-helper
+# Chrome Extension Campaign Monitor Helper
+
 Chrome Extension for disabling "Inline CSS" when importing HTML email into campaign monitor
```
Comment (timatron) [inline README.md]: more details please

File: README.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+~% npm install
+~% npm start
+```
+
+## Install the extension on Chrome/Brave
+
+-   Enter `chrome://extensions/` in Chrome/Brave URL
+-   Activate developer mode
+-   `Load unpacked` then choose the repo
+-   Ready!
```
Comment (timatron) [inline README.md]: maybe use chatgpt to expand the readme

File: content.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * This file is only loaded if the URL satisfies the matched pattern.
+ */
+
+/**
+ * DOMContentLoaded listener won't work because the page is already loaded
+ * CM dynamically pulls in content, so we need to wait for the checkbox to be added
+ */
+
+const checkCheckboxStatus = () => {
+    const checkbox = document.getElementById('move-css-inline-choice');
```
Comment (timatron) [inline content.js:11]: shouldn't we only do this on campaign monitor domains?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>TDU Helper</title>
```
Comment (timatron) [inline index.html]: rename and give full description

File: index.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+        <style>
+            body {
+                width: 200px;
+            }
+        </style>
+    </head>
+    <body>
+        <h3>Happy Sending!</h3>
```
Comment (timatron) [inline index.html]: redo with full explanation of what this does

File: manifest.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Chrome Extension Campaign Monitor Helper",
+    "description": "A TDU extension that unchecks the Move CSS inline option when importing HTML into a campaign.",
```
Comment (timatron) [inline manifest.json]: swap TDU for "The Daily Upside

Comment (timatron) [reply manifest.json]: say "into a campaign in Campaign Monitor"

File: manifest.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    "version": "1.0",
+    "manifest_version": 3,
+    "action": {
+        "default_popup": "index.html",
+        "default_icon": "sunstrisk.png"
+    },
+    "permissions": ["webNavigation"],
+    "content_scripts": [
+        {
+            "matches": ["https://*.createsend.com/campaigns/*"],
```
Comment (timatron) [inline manifest.json:17]: oh great! nevermind the above where i said shouldn't we match the domain :)

File: package.json [hunk 9]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "chrome-extension-campaign-monitor-helper",
+    "version": "1.0.0",
+    "private": true,
+    "description": "A TDU extension that unchecks the Move CSS inline option when importing HTML into a campaign.",
```
Comment (timatron) [inline package.json]: s/TDU/The Daily Upside/ 
s/a campaign/a campaign in Campaign Monitor/

**Review by timatron — APPROVED** [review]
a few comments, but awesome!

File: README.md (hunk 1 above)
Comment (arashsafaridev) [reply README.md]: Yes, if it's published it works like a standard extension (from the url). But to upload an extension manually we need to load it unpacked for now.

File: content.js (hunk 4 above)
Comment (arashsafaridev) [reply content.js:11]: Yes, it's in extensions manifest. This script will run if it matches CM pattern and only on campaign pages.
```json
"content_scripts": [
        {
            "matches": ["https://*.createsend.com/campaigns/*"],
            "js": ["content.js"]
        }
    ]
```

**Pull request merged**